templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -206,6 +207,26 @@ Under the line
 
 ![](/static/images/help/markdown-paragraph.png)
 
+## [Admonitions](#admonitions)
+
+Admonitions are used to provide colorization for a block of test. You will most likely
+use them in the context of bots or webhooks.
+
+Supported statuses are:
+
+* success (green)
+* warning, caution (yellow)
+* danger, error, critical (red)
+* info, primary (blue)
+* neutral, secondary (gray)
```

Comment:
What do "primary" and "secondary" mean? I'm not sure they fit with "info" and "neutral".

Reply:
That's just a bootstrap-ism. It needs improvement (or elimination).